akka-docs/src/main/paradox/java/typed.md

```diff
@@ -9,16 +9,49 @@ This module is currently marked as @ref:[may change](common/may-change.md) in th
 
 @@@
 
-As discussed in `actor-systems` (and following chapters) Actors are about
+## Dependency
+
+Akka Typed APIs are bundled in the `akka-typed` artifact.
+Make sure that you have the following dependency in your project:
+
+Scala
+:   @@@vars
+    ```
+    "com.typesafe.akka" %% "akka-typed" % "$akka.version$"
+    ```
+    @@@
+
+Java
```

Comment:
Scala (sbt) / Java (maven) - note that we say sbt/maven/gradle on http://akka.io/docs/ so we should start converging towards that :)

Should we also add a Gradle tab?